core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -997,6 +1049,26 @@ public static int lengthOf(@Nullable CharSequence columnValue) {
         return columnValue != null ? columnValue.length() : NULL_LEN;
     }
 
+    public static @NotNull MatViewDefinition loadMatViewDefinition(
```

Comment:
This is used in 1 place only. Move the load method into Mat View Graph to not polute Table Utils with non-primitive dependencies

Reply:
moved to MatViewDefinition